clouddriver-kubernetes-v2/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/provider/KubernetesV2InstanceProvider.java

```diff
@@ -82,49 +85,76 @@ public KubernetesV2Instance getInstance(String account, String location, String
   }
 
   @Override
-  public List<ContainerLog> getConsoleOutput(String account, String location, String fullName) {
-    Optional<KubernetesV2Credentials> optionalCredentials = accountResolver.getCredentials(account);
+  public List<ContainerLog> getConsoleOutput(String account, String namespace, String fullName) {
+    final Optional<KubernetesV2Credentials> optionalCredentials =
```

Comment:
We don't tend to mark local variables as `final` in the Spinnaker codebase, though I suppose this is more of a personal preference thing.  We definitely *do* mark fields `final` wherever possible, but in the case of local variables in short methods I think the benefit of the compiler preventing reassignment is not really outweighed by the visual noise.  That being said, I know opinions differ on this, so either way is fine.

Reply:
I think its worth matching the rest of the codebase, I'll update this tomorrow morning